apps/editing-toolkit/package.json

```diff
@@ -126,6 +127,7 @@
 		"@testing-library/jest-dom": "^5.16.2",
 		"@testing-library/react": "^12.1.3",
 		"@types/node": "^18.11.18",
+		"@types/wordpress__editor": "^13.6.0",
```

Comment:
Note that those types are still not loaded due to a forced noop package resolution here:

https://github.com/Automattic/wp-calypso/blob/8c09604f5f03362894e1ac1a947117192453d13e/package.json#L293